internal/common/jsonofflinestorage/jsonofflinestorage.go

```diff
@@ -165,3 +113,29 @@ func (l *JSONFileStorage) Restore() error {
 	slog.Info("Restored from file", "file", l.fileName)
 	return nil
 }
+
+type SavedChan = chan struct{}
+
+func (l *JSONFileStorage) SaverRoutine(saved SavedChan) error {
+	t := time.NewTimer(l.storeInterval)
+	t.Stop()
```

Comment:
То же самое - достаточно запутанная логика использования таймера.
```suggestion
	defer t.Stop()
```

Reply:
++